src/fidesops/ops/service/privacy_request/request_runner_service.py

```diff
@@ -198,10 +213,10 @@ def run_privacy_request(
         privacy_request = PrivacyRequest.get(db=session, object_id=privacy_request_id)
         if privacy_request.status == PrivacyRequestStatus.canceled:
             logging.info(
-                f"Terminating privacy request {privacy_request.id}: request canceled."
+                "Terminating privacy request %s}: request canceled.", privacy_request.id
```

Comment:
missed }

Reply:
ah thanks, I tried to do a search too but missed these